Update mapd-draw dependency to latest commit

vastcharade opened the pull request:
Prior, the commit the mapd-draw dependency was pointing to was pre-rebranding efforts, so to move to a top-of-branch commit meant we needed to update all the mapd-draw-related import statements to reflect the rebrand-related naming. So paths such as `@heavyai/draw/dist/mapd-draw` needed to be updated to `@heavyai/draw/dist/draw`

# Merge Checklist
## :wrench: Issue(s) fixed:
- [ ] Author referenced issue(s) fixed by this PR:
- [ ] Fixes #0

## :smoking: Smoke Test
- [ ] Works in chrome
- [ ] Works in firefox
- [ ] Works in safari
- [ ] Works in ie edge
- [ ] Works in ie 11

## :ship: Merge
- [ ] author crafted PR's title into release-worthy commit message.

johallar reviewed (approved):
:tada: Awesome! I just noticed we had this package duped yesterday as well

vastcharade commented:
Made a quick change as the tests highlighted that pulling in `HeavyConnect` as a dependency in `raster-chart.js` was breaking builds. So I removed that dependency and updated the build. @johallar you may want to take a quick look at that commit. Unfortunately I have to wait until the top of the hour for the checks to run :(

vastcharade commented on src/charts/raster-chart.js:
```diff
@@ -697,18 +696,18 @@ export default function rasterChart(parent, useMap, chartGroup, _mapboxgl) {
     callback,
     fetchEvenIfEmpty = false
   ) {
+    if (!point) {
```
early out should be done up-front.

vastcharade commented on src/charts/raster-chart.js:
```diff
@@ -697,18 +696,18 @@ export default function rasterChart(parent, useMap, chartGroup, _mapboxgl) {
         : _chart.height()
     const pixelRatio = _chart._getPixelRatio() || 1
-    const pixel = new HeavyConnect.TPixel({
+    const pixel = {
```
can just set the pixel = `{x: ..., y: ...}` - this will get properly converted to a `TPixel` in the `getResultRowForPixelAsync` call below. See: https://github.com/heavyai/heavyai-connector/blob/master/src/heavy-con-es6.js#L1803

johallar commented:
Changes LGTM
